TSRM/tsrm_win32.c

```diff
@@ -688,7 +688,20 @@ TSRM_API int shmget(key_t key, size_t size, int flags)
 		CloseHandle(shm_handle);
 		return -1;
 	}
-	shm->segment = shm_handle;
+	if (NULL == shm->segment || shm->segment == INVALID_HANDLE_VALUE) {
```

Comment:
nit: kind of same remark, might be best to be consistent with left operand vs right operand place of choice.

Reply:
I don't mind.